independent-publisher-connectors/Microsoft Learn Catalog/apiDefinition.swagger.json

```diff
@@ -0,0 +1,780 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Learn Catalog",
+    "description": "A connector to the Microsoft Learn Catalog",
```

Comment:
@SeanKe11y thank you so much for the quick fix. I would highly recommend you give a proper description for your connector here in the swagger file. this is what will show in your connector documentation. I suggest you use the same description in the readme file, and just remove the Word "API".

Reply:
@Amjed-Ayoub, good point, I've made that change to the swagger file, see latest commit.